internal/pkg/aws/sessions/errors.go

```diff
@@ -39,16 +39,11 @@ func (e *errCredRetrieval) Error() string {
 // RecommendActions returns recommended actions to be taken after the error.
 // Implements main.actionRecommender interface.
 func (e *errCredRetrieval) RecommendActions() string {
-	notice := "It looks like your credential settings are misconfigured or missing"
+	notice := "It looks like AWS credentials are misconfigured or missing"
```

Comment:
```suggestion
	notice := "It looks like your AWS credentials are misconfigured or missing"
```

Reply:
curious, why your is needed?

Reply:
So the user knows it's their issue to fix rather than something internal.